docs/apache-airflow/howto/docker-compose/index.rst

```diff
@@ -149,7 +149,7 @@ On **all operating systems**, you need to run database migrations and create the
 
 .. code-block:: bash
 
-    docker compose up airflow-init
+    docker compose run airflow-init
```

Comment:
[`docker compose run`](https://docs.docker.com/engine/reference/commandline/compose_run/) has a different purpose (run a command) if compare to the [`docker compose up`](https://docs.docker.com/engine/reference/commandline/compose_up/) (run a service)

There is no any problem to the start container with up command at least in the CI and docker-compose sample tests as well as in Linux/MacOS, in addition this service would also use later for services dependencies check.

Reply:
I'm running this on Windows 11 with Docker Desktop and `docker compose up airflow-init` does not initialize the DB, but `docker compose run airflow-init` does.

> in addition this service would also use later for services dependencies check.
That will be run anyway by the depends_on of the webserver and scheduler, so my change in the doc doesn't influence that.  
The doc specifically mentions to initialize the DB "Before starting Airflow for the first time", so this is a one-time manual process.

Reply:
I am not against changing it to run.

It quite matches the purpose of run vs. up as described in https://docs.docker.com/compose/faq/#whats-the-difference-between-up-run-and-start

But I think you'd ne need to investigate why `up` does not work for you. It should. Can you explain HOW it does not work for you? What happens?

However `run` has some properties that are different than `up` - for example it will leave the container behind when you use run, so generally you should also use `--rm` flag if you want to use run becaause (unlike up) it will create a new container every time you run it , where `up` will use the same project as the main project you use when you start other services in it.

Also any change here should be acompanied by updating our tests.

 We are running `docker compose` quick start tests in our CI 

https://github.com/apache/airflow/blob/main/.github/workflows/ci.yml#L1929C34-L1929C34

and thanks to that both me and @Taragolis believe that it works - because it passes the tests (and that's why we are asking  you why it does not work for you and what problems does it solve).

Maybe there is a good reason it does not work for your Windows 11 desktop and it would be good if you could investigate why.

And yes - maybe the conclusion is that it should be changed to use `run` but if we do, then this PR should also add similar step to our docker-compose tests to make sure that what we have in our docs is also tested in our CI.,